src/sentry/sentry_metrics/indexer/cache.py

```diff
@@ -37,63 +39,60 @@ def randomized_ttl(self) -> int:
         jitter = random.uniform(0, 0.25) * cache_ttl
         return int(cache_ttl + jitter)
 
-    def make_cache_key(self, key: str, cache_namespace: str) -> str:
-        hashed = md5_text(key).hexdigest()
-        return f"indexer:{self.partition_key}:org:str:{cache_namespace}:{hashed}"
+    def make_cache_key(self, key: str) -> str:
+        # slow rollout option will be used here
```

Comment:
nit: we'll probably want to get rid of this comment since we're taking the double-write approach

Reply:
fixed